modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -184,27 +184,24 @@
 		<method name="range" qualifiers="vararg">
 			<return type="Array" />
 			<description>
-				Returns an array with the given range. Range can be 1 argument [code]N[/code] (0 to [code]N[/code] - 1), two arguments ([code]initial[/code], [code]final - 1[/code]) or three arguments ([code]initial[/code], [code]final - 1[/code], [code]increment[/code]). Returns an empty array if the range isn't valid (e.g. [code]range(2, 5, -1)[/code] or [code]range(5, 5, 1)[/code]).
-				Returns an array with the given range. [code]range()[/code] can have 1 argument N ([code]0[/code] to [code]N - 1[/code]), two arguments ([code]initial[/code], [code]final - 1[/code]) or three arguments ([code]initial[/code], [code]final - 1[/code], [code]increment[/code]). [code]increment[/code] can be negative. If [code]increment[/code] is negative, [code]final - 1[/code] will become [code]final + 1[/code]. Also, the initial value must be greater than the final value for the loop to run.
-				[code]range()(/code] converts all arguments to [int] before processing.
+				Returns an array with the given range. The [b]range()[/b] function can be called in three ways:
+				[b]range ([/b] int [code]n[/code] [b])[/b]: Starts from 0, changes by steps of 1, and stops [i]before[/i] [code]n[/code]. Variable [code]n[/code] is exclusive.
+				[b]range ([/b] int [code]b[/code], int [code]n[/code] [b])[/b]: Starts from [code]b[/code], changes by steps of 1, and stops [i]before[/i] [code]n[/code]. Variables [code]b[/code] and [code]n[/code] are [b]inclusive[/b] and [b]exclusive[/b], respectively.
+				[b]range ([/b] int [code]b[/code], int [code]n[/code], int [code]s[/code] [b])[/b]: Starts from [code]b[/code], changes by steps of [code]s[/code], and stops [i]before[/i] [code]n[/code]. Variables [code]b[/code] and [code]n[/code] are [b]inclusive[/b] and [b]exclusive[/b], respectively. Variable [code]s[/code] [b]can[/b] be negative. Raise an error if [code]s[/code] is zero.
+				[b]range()[/b] converts all arguments to [int] before processing.
```

Comment:
```suggestion
				[method range] converts all arguments to [int] before processing.
```

Reply:
Done. However #60394 has already provided a fix for this line, should I discard this change from the commit?